test/core/SedaCoreV1.test.ts

```diff
@@ -102,36 +102,77 @@ describe('SedaCoreV1', () => {
 
     const data = { requests, results, proofs, wallets, initialBatch, validatorProofs };
 
-    return { prover, core, data };
+    return { prover, core, feeManager, data };
   }
 
-  describe('getPendingRequests', () => {
-    it('returns correct requests with pagination', async () => {
+  describe('postRequest', () => {
+    it('allows duplicate request submissions but does not repost', async () => {
       const { core, data } = await loadFixture(deployCoreFixture);
```

Comment:
This test doesn't align with core contract right? We allow people to do this on the core contract?

Reply:
Yes, but only if it is not resolved. The idea is to deter front-running the `postRequest` (with low fees) because the legit `postRequest` will not "repost" (most probably wrong wording) but simply increase fees.